cookbooks/aws-parallelcluster-slurm/recipes/config/config_head_node.rb

```diff
@@ -282,6 +283,21 @@
   retry_delay 2
 end unless redhat_on_docker?
 
+template "#{node['cluster']['scripts_dir']}/slurm/check_login_nodes_stopped.sh" do
+  source 'slurm/head_node/check_login_nodes_stopped.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0700'
+  variables(
+    cluster_name: node['cluster']['cluster_name'] || node['cluster']['stack_name'],
+    login_nodes_pool_name: lazy { node['cluster']['config'].dig(:LoginNodes, :Pools, 0, :Name) },
+    region: node['cluster']['region']
+  )
+  only_if do
+    node['cluster']['config'].dig(:LoginNodes)
+  end
+end
+
```

Comment:
Since we are re-using this block also in `update_head_node.rb`, can we move it to a sub-recipe that we call from both `config_head_node.rb` and `update_head_node.rb`?

Reply:
Also, ideally we could add unit tests (ChefSpec tests) for the sub-recipe. But we can leave it for later.

Reply:
Done!